src/pages/Login/CreateAccountFreeTrial.js

```diff
@@ -0,0 +1,141 @@
+import React from 'react'
+import { Link } from 'react-router-dom'
+import { PATHS } from '../../App'
+import withSizes from 'react-sizes'
+import LoginEmailForm, { EmailForm } from './LoginEmailForm'
+import cx from 'classnames'
+import Panel from '@santiment-network/ui/Panel'
+import MobileWrapper from './Mobile/MobileWrapper'
+import { mapSizesToProps } from '../../utils/withSizes'
+import SwipablePages from '../../components/SwipablePages/SwipablePages'
+import externalStyles from './index.module.scss'
+import styles from './CreateAccountFreeTrial.module.scss'
+
+const TRIAL_DESCRIPTIONS = [
+  {
+    text:
+      'Full historical and present data for 30+ on-chain, social & development metrics'
+  },
+  {
+    text:
+      'Exclusive End-of-Month market & network reports from the Santiment team'
+  },
+  {
+    text:
+      'Complete access to our Spreadsheets plugin and our internal market templates'
+  }
+]
+
+const TrialDescriptions = () => {
+  return (
+    <div className={styles.trials}>
+      {TRIAL_DESCRIPTIONS.map(({ text }, index) => {
+        return (
+          <div className={styles.trial} key={index}>
+            <svg
+              className={styles.icon}
+              width='17'
+              height='19'
+              viewBox='0 0 17 19'
+              fill='none'
+              xmlns='http://www.w3.org/2000/svg'
+            >
+              <path
+                d='M6.74023 9.73535C6.77366 9.75209 6.79036 9.76045 6.81543 9.76881C6.79036 9.76045 6.78201 9.75209 6.74023 9.73535Z'
+                fill='#181B2B'
```

Comment:
night mode - ok? I think, should be dynamic color

Reply:
@haritonasty 
![image](https://user-images.githubusercontent.com/14061779/75873461-00e28b80-5e21-11ea-885e-4c26851b00b9.png)

Reply:
yep, use color variable for bullets